src/plugins/intel_gna/src/gna_plugin.cpp

```diff
@@ -1127,43 +1173,36 @@ uint32_t GNAPlugin::QueueInference(const InferenceEngine::BlobMap& inputs, Infer
         auto importedElementSizeBytes = gnaFlags->sw_fp32 ? 4 : (gnaFlags->input_low_precision ? 1 : 2);
         auto importedBytes = importedElements * importedFrames * importedElementSizeBytes;
 
-        if (inputs_ptr_->at(input.first).get_required_size() < importedBytes) {
-            THROW_GNA_EXCEPTION << "Cannot import input frames for :" << input.first
-                                << ", allocated size: " << inputs_ptr_->at(input.first).get_required_size()
+        if (inputs_ptr_->at(input_name).get_required_size() < importedBytes) {
+            THROW_GNA_EXCEPTION << "Cannot import input frames for :" << input_name
+                                << ", allocated size: " << inputs_ptr_->at(input_name).get_required_size()
                                 << ", but input blob size: " << importedBytes;
         }
 
-        ImportFrames(inputs_ptr_->at(input.first).ptrs[index],
-                     input.second->cbuffer().as<float*>(),
-                     input.second->getTensorDesc().getPrecision(),
-                     gnaFlags->sw_fp32 ? kScaleFactorDefault : inputs_ptr_->at(input.first).scale_factor,
+        // Perform preprocessing on CPU
+        std::shared_ptr<ov::Model> model = inputs_ptr_->at(input_name).pre_post_process_model;
+        if (model != nullptr) {
```

Comment:
If we replace `if (transpose_info != std::end(transpose_inputs_info)) {`
with `if (model != nullptr) {` 
why do we still need to have  transpose_inputs_info field ?

Reply:
transpose_info will be removed at all  later